api/src/main/java/marquez/service/EventService.java

```diff
@@ -0,0 +1,10 @@
+package marquez.service;
+
+import lombok.NonNull;
+import marquez.db.BaseDao;
+
+public class EventService extends DelegatingDaos.DelegatingEventDao {
```

Comment:
Though the Marquez codebase uses services throughout (thanks for following our convention), we are moving away from defining services as delegating objects in favor of using the DAOs directly if no business logic is defined. You can remove this call all together to simplify things! ;)

Reply:
Done.